iphone/Classes/TiUIListView.m

```diff
@@ -2004,7 +1993,7 @@ - (void)searchBarTextDidBeginEditing:(UISearchBar *)searchBar
     }
     self.searchString = (searchBar.text == nil) ? @"" : searchBar.text;
     [self buildResultsForSearchText];
-    [[searchController searchResultsTableView] reloadData];
+    [_tableView reloadData];
```

Comment:
`_tableView` is the main table view, but the deleted line reloaded the search-results table view.